docs/AboutUs.md

```diff
@@ -15,14 +15,23 @@ We are a team based in the [School of Computing, National University of Singapor
 [[github](https://github.com/huizhuansam)]
 [[portfolio](team/johndoe.md)]
 
-* Role: Scheduling and tracking
-* Responsibilities: User assistance
+- Role: Scheduling and tracking
```

Comment:
@zhenghanlee help me add `deliverables and deadlines` thanks!

Reply:
> @zhenghanlee help me add `deliverables and deadlines` thanks!

You mean for `Role`?

Reply:
The TP docs put them as separate roles:
"_Deliverables and Deadlines_: Ensure project deliverables are done on time and in the right format."
"_Scheduling and Tracking_: In charge of defining, assigning, and tracking project tasks."
So I think there is a clear difference.